tasks/run-matlab-tests/v0/main.ts

```diff
@@ -13,7 +13,10 @@ async function run() {
             CoberturaCodeCoverage: taskLib.getInput("codeCoverageCobertura"),
             SourceFolder: taskLib.getInput("sourceFolder"),
             SelectByFolder: taskLib.getInput("selectByFolder"),
-            SelectByTag: taskLib.getInput("selectByTag")};
+            SelectByTag: taskLib.getInput("selectByTag"),
+            CoberturaModelCoverage: taskLib.getInput("coberturaModelCoverage"),
```

Comment:
Should this be `modelCoverageCobertura` to match `codeCoverageCobertura`?

Reply:
Done.